test/scenarios/run.js

```diff
@@ -257,18 +258,21 @@ async function runRemote(scenario, helpers) {
       scenario,
       helpers.pouch,
       helpers.local.syncDir.abspath,
-      relpathFix
+      relpathFix,
+      useRealInodes
     )
     await helpers.remote.ignorePreviousChanges()
   }
 
   await remoteCaptureHelpers.runActions(scenario, cozyHelpers.cozy)
 
+  await helpers.local.side.watcher.start()
   await helpers.remote.pullChanges()
   // TODO: Don't sync when scenario doesn't have target FS/trash assertions?
-  for (let i = 0; i < scenario.actions.length + 1; i++) {
-    await helpers.syncAll()
-  }
+  await helpers.syncAll()
+  // Wait for all local events to be dispatched
+  await helpers.local.simulateAtomEvents([])
```

Comment:
No guarantee that the magic event is not pushed before the FS ones.

Reply:
All tests are green but since this PR is about tests lying I'll add a wait time before sending the magic event.
500 ms should be enough.